content/bn/docs/reference/glossary/workload.md

```diff
@@ -0,0 +1,19 @@
+---
+title: ওয়ার্কলোড
+id: workload
+date: 2019-02-13
+full_link: /docs/concepts/workloads/
+short_description: >
+   ওয়ার্কলোড হলো কুবারনেটিস-এ চলমান একটি অ্যাপ্লিকেশন।
+
+aka: 
+tags:
+- fundamental
+---
+   ওয়ার্কলোড(workload) হলো কুবারনেটিস-এ চলমান একটি অ্যাপ্লিকেশন।
```

Comment:
ওয়ার্কলোড হলো কুবারনেটিস-এ চলমান একটি অ্যাপ্লিকেশন।

Reply:
I fixed the issue